ui/e2e/nightwatch.json

```diff
@@ -0,0 +1,67 @@
+{
```

Comment:
Could we add some quick docs for us non-UI folks to understand what this does?  i.e. how nightwatch, etc all work together.  You don't have to go too deep we can RTFM just a high level.